tests/ballerina-unit-test/src/test/resources/test-src/expressions/builtinoperations/clone-operation-negative.bal

```diff
@@ -0,0 +1,9 @@
+public function cloneWithParams() {
```

Comment:
Shall we add the license header to this file?

Reply:
Fixed